app/src/main/java/com/eveningoutpost/dexdrip/utils/Preferences.java

```diff
@@ -268,6 +268,7 @@ public void onCreate(Bundle savedInstanceState) {
             addPreferencesFromResource(R.xml.pref_community_help);
 
             bindTTSListener();
+            bindBgMissedAlertsListener();
```

Comment:
be aware that this will bypass the check on alerts_disabled_until.
Actualy in AlertType a check is done on alerts_disabled_until , if this is higher than now, then you see that there's no call to checkIfMissedReadingAlert https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/blob/master/app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java#L104

If you now set this listener for preference change, a user may have all alerts disabled, but change the preference for missed readings, then that alert will not be suppressed anymore.

Normally get_highest_active_alert should be called every 5 minutes https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/blob/master/app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java#L102 so i don't think it's necessary that have that listener